docs/tutorials/fundamentals/tutorial-obtain-token-and-purchase-ram.md

```diff
@@ -0,0 +1,125 @@
+---
+title: 'Tutorial - Using the Faucet and Buying RAM on Ultra Testnet'
+order: -99996
+outline: [0, 5]
+---
+
+# Tutorial - Using the Faucet and Buying RAM on Ultra Testnet
+
+This tutorial will guide you through the process of obtaining UOS tokens using the faucet and purchasing RAM on the Ultra blockchain testnet. These actions are crucial for participating in blockchain activities, such as deploying contracts or storing unique digital assets (Uniqs).
+
+## Prerequisites
+
+Before starting this tutorial, ensure you have the following:
+
+- **A Developer Testnet Account**: You need an account on the Ultra testnet. If you don't have one, please refer to our previous tutorial on [how to create a testnet account](./tutorial-generate-key-and-create-testnet-account.md).
+- **Private and Public Keys**: Your account’s keys should be securely stored. You’ll need your private key for signing transactions during this tutorial.
+
+## Goal
+
+The goal of this tutorial is two-fold:
+1. **To add UOS tokens to your testnet account using the faucet**: The faucet is a tool that distributes free UOS tokens on the testnet, allowing you to perform transactions and interact with the blockchain without real financial cost.
+2. **To purchase RAM for your account using `eosio.buyram`**: RAM in the EOSIO ecosystem is needed to store data on the blockchain. For example, if you want to store or manage digital assets like Uniqs, you need sufficient RAM. This tutorial will show you how to increase your RAM quota based on current prices and the amount of UOS you are willing to spend.
+
+By the end of this tutorial, you should be able to comfortably manage your resources on the Ultra blockchain, paving the way for more complex interactions such as deploying smart contracts or trading digital assets.
+
+## Obtaining UOS Tokens Using the Faucet
+
+This section will guide you through the process of obtaining UOS tokens on the Ultra testnet. This is crucial for performing transactions and interacting with various blockchain functionalities.
+
+### Step 1: Check Initial Balance
+
+Before obtaining additional UOS tokens, first check your current balance to verify the amount of UOS you have available. Visit the Ultra Blockchain Explorer at [https://explorer.testnet.ultra.io](https://explorer.testnet.ultra.io) and search for your account (`1aa2aa3aa4in`). The initial balance (8.99999999 UOS) will be displayed.
+
+![](./images/faucet-initial-uos-balance.png)
+
+### Step 2: Request UOS Tokens from the Faucet
+
+1. Open the faucet page at [https://faucet.testnet.app.ultra.io](https://faucet.testnet.app.ultra.io).
```

Comment:
There is no way to force it to open the `Token Faucet` tab specifically, it may open the `Account Creator` instead. so you need to add an extra step and screenshot showing how to navigate to this tab.